tests/clpy_tests/math_tests/test_sumprod.py

```diff
@@ -239,7 +243,13 @@ def test_cumprod_2dim_with_axis(self, xp, dtype):
 
     @testing.slow
     def test_cumprod_huge_array(self):
-        size = 2 ** 32
+        device_max_alloc_bytes =\
+            clpy.backend.opencl.utility.\
+            GetDeviceMaxMemoryAllocation(
+                clpy.backend.opencl.env.get_device_id()
```

Comment:
Why don' you get `cl_device_id` directly via `env.get_device()` like other APIs?

Reply:
When I set the argument type to `cl_device_id` like the others, Cython compiler said that `can't convert python object to cl_device_id`.
This message was caused by the difference of `cdef` and `cpdef` (intended to provide pure python API).

If the argument type must be `cl_device_id`, another wrapper dedicated to testing will be needed.
I thought it'll be a bit redundant.